source/src/notebook/healthcare-and-life-sciences/molecular-unfolding/molecular_unfolding.ipynb

```diff
@@ -0,0 +1,891 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    # print(f"dwave-qa run time {qa_optimize_result['time']}")
</pre><p>should we remove it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/quantum-computing-exploration-for-drug-discovery-on-aws/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='29'/>

Reply:
<p>already remove this line</p>